backend/src/interfaces.ts

```diff
@@ -17,7 +17,7 @@ export interface Level {
   locationName: string | undefined;
   latitude: number;
   longitude: number;
-  zPosition: number | undefined;
+  zPosition: string | undefined;
```

Comment:
maybe what would make sense here is a number that represents numerically what floor they're on (if applicable). for example:
LG -> 0
G   ->  1
1   ->  2
and so on. this will be quite useful for determining how far off a guess is in terms of floors.
like if someone guessed room 101 when the answer is 301, assuming they're literally on top of each other, we should give a guess for room 201 a higher score than 101, and 301 the perfect score. does that make sense?
see if you can implement this when you're pushing to the `levels` array